# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xterm/login-shell.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xterm-404/charproc.c.vanilla	2025-12-04 19:09:51.262886541 +0200
+++ xterm-404/charproc.c	2025-12-04 19:10:03.487000000 +0200
@@ -450,7 +450,7 @@
     Bres(XtNjumpScroll, XtCJumpScroll, screen.jumpscroll, True),
     Bres(XtNkeepClipboard, XtCKeepClipboard, screen.keepClipboard, False),
     Bres(XtNkeepSelection, XtCKeepSelection, screen.keepSelection, True),
-    Bres(XtNloginShell, XtCLoginShell, misc.login_shell, False),
+    Bres(XtNloginShell, XtCLoginShell, misc.login_shell, True),
     Bres(XtNmarginBell, XtCMarginBell, screen.marginbell, False),
     Bres(XtNmetaSendsEscape, XtCMetaSendsEscape, screen.meta_sends_esc, DEF_META_SENDS_ESC),
     Bres(XtNmultiScroll, XtCMultiScroll, screen.multiscroll, False),
